src/Everscale/Address.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2017-2021 Trust Wallet.
```

Comment:
New added files copyright should be `2022` now
```suggestion
// Copyright © 2017-2022 Trust Wallet.
```